examples/shapes_and_collections/fancybox_demo.py

```diff
@@ -19,8 +19,8 @@
 styles = mpatch.BoxStyle.get_styles()
 ncol = 2
 nrow = (len(styles) + 1) // ncol
-axs = (plt.figure(figsize=(3 * ncol, 1 + nrow))
-       .add_gridspec(1 + nrow, ncol, wspace=.5).subplots())
```

Comment:
fwiw I prefer add_gridspec to gridspec_kw...

Reply:
I know, but I don't see how that improves this example since it's not about `add_gridspec`.